src/Microsoft.AspNetCore.Hosting/Startup/ExceptionPage/Views/ErrorPage.js

```diff
@@ -0,0 +1,192 @@
+﻿(function (window, undefined) {
```

Comment:
:cry: could we not copy in all of the terribleness from Diagnostics? This is a mini jquery impl and we don't need all of it.